tests/cupyx_tests/scipy_tests/special_tests/test_ufunc_dispatch.py

```diff
@@ -0,0 +1,37 @@
+import numpy
+import cupy
+import scipy.special
+import cupyx.scipy.special
+
+from cupy import testing
+import pytest
+
+scipy_ufuncs = {
+    f
+    for f in scipy.special.__all__
+    if isinstance(getattr(scipy.special, f), numpy.ufunc)
+}
+cupyx_scipy_ufuncs = {
+    f
+    for f in dir(cupyx.scipy.special)
+    if isinstance(getattr(cupyx.scipy.special, f), cupy.ufunc)
+}
+
+
+@testing.gpu
+@testing.with_requires("scipy")
+@pytest.mark.parametrize("ufunc", cupyx_scipy_ufuncs & scipy_ufuncs)
```

Comment:
It might not work with pytest-xdist because sets are unordered.

Reply:
Interesting! I verified that it doesn't work.
With `sorted(cupyx_scipy_ufuncs & scipy_ufuncs)` pytest-xdist works (tried with `python -m pytest -n auto tests/cupyx_tests/scipy_tests/special_tests/test_ufunc_dispatch.py`).